INFO     common.recordlog:conftest.py:23 -----------------接口测试开始---------------------
INFO     common.recordlog:sendrequests.py:32 接口名称：删除用户
INFO     common.recordlog:sendrequests.py:33 接口请求地址：http://127.0.0.1:8787/dar/user/deleteUser
INFO     common.recordlog:sendrequests.py:34 测试用例名称：无效删除用户
INFO     common.recordlog:sendrequests.py:35 请求头：{'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8'}
INFO     common.recordlog:sendrequests.py:36 cookies：None
INFO     common.recordlog:sendrequests.py:42 请求参数：{'data': {'user_id': 1212718219298}}
INFO     common.recordlog:sendrequests.py:26 接口返回信息：{
  "msg": "删除失败，用户id不存在!",
  "msg_code": 9001
}

INFO     common.recordlog:assertions.py:90 yaml文件的预期结果：[{'contains': {'msg': '删除失败'}}]
INFO     common.recordlog:assertions.py:39 字符串包含断言成功：预期结果-->删除失败；实际结果-->删除失败，用户id不存在!
INFO     common.recordlog:assertions.py:109 测试成功
INFO     common.recordlog:conftest.py:25 -----------------接口测试结束---------------------